vmm/src/cpu.rs

```diff
@@ -685,10 +685,21 @@ impl CpuManager {
                 .map_err(Error::CpuidSgx)?;
         }
 
-        // Set CPU physical bits
+        // Update some existing CPUID
         for entry in cpuid.as_mut_slice().iter_mut() {
-            if entry.function == 0x8000_0008 {
-                entry.eax = (entry.eax & 0xffff_ff00) | (phys_bits as u32 & 0xff);
+            match entry.function {
+                // Set CPU physical bits
+                0x8000_0008 => {
+                    entry.eax = (entry.eax & 0xffff_ff00) | (phys_bits as u32 & 0xff);
+                }
+                // Disable KVM_FEATURE_ASYNC_PF_INT
+                // This is required until we find out why the asynchronous page
+                // fault is generating unexpected behavior when using interrupt
+                // mechanism.
+                0x4000_0001 => {
+                    entry.eax &= 0xffff_bfff;
```

Comment:
I would rather write this as
`entry.eax &= !(1 << KVM_FEATURE_ASYNC_PF_INT_BIT)` or whatever the kernel or QEMU uses for maximum googlability since we will forget :-)

Reply:
Good point :)